scripts/fast-reboot

```diff
@@ -106,6 +106,18 @@ function clear_warm_boot()
     fi
 }
 
+function init_warm_reboot_states()
```

Comment:
>init_warm_reboot_states [](start = 9, length = 23)

You could move code into backup_database(). And consider to adapt the optimization there.

Reply:
That would interfere the warm reboot. Syncd warmshutdown uses this table to coordinate operations. Some module (not today, but maybe in the future) want to leave information to be handled after warm reboot is done. So it would be best to clear the table before any real warm reboot operation started.

Reply:
Incorporated the optimization. Thanks!